source/includes/_objects.md

```diff
@@ -268,6 +300,34 @@ gui.close();
 
 These very simple methods open and close the gui, and neither take any arguments.
 
+# Inventory
+
+The Inventory object contains methods used for getting information about the user's inventory. It can be called
+through `Player.getInventory()`. More about the `Player` class can be found [later](#player).
+
+<aside class="warning">Looping through the inventory can be an intensive process if done rapidly.</aside>
```

Comment:
Why is this warning needed? I haven't used the inventory API in a while (or maybe every), but at a glance it doesn't seem more intensive than anything else.